cpp/include/gpu_operations.h

```diff
@@ -95,7 +115,16 @@ class GpuOperations {
     return h_c;
   }
 
-  static Matrix<T> Multiply(const Matrix<T> &a, const Matrix<T> &b) {
+  /// This function multiplies an input Matrix and a Matrix
```

Comment:
I think this was to make sure that matrix vector version of the Eigen function was running rather than the matrix matrix version for testing. It is uncommented in the next push.